plugins/analogix/fu-analogix-common.c

```diff
@@ -0,0 +1,231 @@
+/*
+ * Copyright (C) 2021 Xiaotian Cui <xtcui@analogixsemi.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+#include "fu-analogix-common.h"
+
+guint64
+hex_str_to_dec (const gchar *str, guint8 len)

```

Comment:
this should be a static function declaration, and the symbol name should have a proper namespace for the plugin